src/js/models/home.js

```diff
@@ -76,5 +97,15 @@ export default class Home{
 			alert("seniorGetOpps: " + error);
 		}
 	}
+
+	async getJuniorInfo(juniorUid){
+		try{			
+			await db.collection('Profiles').doc(juniorUid).get().then(doc => {
+				this.juniorInfo = doc;

```

Comment:
If we changed this to have a return value instead of passing everything through state, we could have one `getProfile` function instead of having multiple (`getHomeData` and `getJuniorInfo` are the exact same function)

Reply:
We can talk more about this later.

Reply:
Created issue